docs/static_site/src/_includes/get_started/windows/python/gpu/pip.md

```diff
@@ -64,7 +64,7 @@ $ pip install mxnet-cu80==0.11.0
 <div class="master">
 
 {% highlight bash %}
-$ pip install mxnet-cu101 --pre
+$ pip install --pre mxnet-cu102 -f https://dist.mxnet.io/python/all
```

Comment:
is cu10.2 upgrade here intended?

Reply:
I think in nightly build we already supported the latest cuda. @leezu please confirm.

Reply:
I think it's reasonable to update to cu10.2. Cu10.2 fixes cuda bugs in cu10.1, so we should recommend 10.2